src/parser/mod.rs

```diff
@@ -6682,6 +6694,14 @@ impl<'a> Parser<'a> {
         Ok(Tag::new(name, value))
     }
 
+    pub(crate) fn parse_rename_object_def(&mut self) -> Result<RenameObjectDef, ParserError> {
```

Comment:
```suggestion
    fn parse_rename_object_def(&mut self) -> Result<RenameObjectDef, ParserError> {
```
maybe better since this method is small and only used once, we can inline it into the callsite instead of having it as a standalone method? e.g.
```rust
let operations = self.parse_comma_separated(|p| {
  let old_name = self.parse_object_name(false)?;
  self.expect_keyword(Keyword::TO)?;
  let new_name = self.parse_object_name(false)?;
  Ok((old_name, new_name))
})?;
```

Reply:
Good idea! Have Done.